frontend/src/utilsLogic.tsx

```diff
@@ -151,6 +152,13 @@ export const makeGuiInstance = (
   const { guiEnabled, persistent, environmentType } =
     (environmentList ?? [])[0] ?? {};
 
+  let myDrive = '';
+  if (
+    environmentType === EnvironmentType.Standalone ||
+    environmentType === EnvironmentType.Container
+  ) {
+    myDrive = status?.myDriveUrl ?? '';
+  }
```

Comment:
Is this actually needed? In case the environment is VM-based, the `myDriveUrl` is always empty (same below).

Reply:
you're right